kratos/python_scripts/python_solver.py

```diff
@@ -187,6 +187,22 @@ def _ImportModelPart(self, model_part, model_part_import_settings):
                 KratosMultiphysics.ReorderAndOptimizeModelPartProcess(model_part, tmp).Execute()
             KratosMultiphysics.Logger.PrintInfo("::[PythonSolver]::", "Finished reading model part from mdpa file.")
 
+        elif (input_type == "json")
```

Comment:
can you please use a more descriptive name?
e.g. `cad_json`, `brep_json` or sth like this? 

In theory we also have a json-IO for meshes, but none uses it

Reply:
I made it to `cad.json` otherwise it wouldn't be recognized as json file in other routines.